src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './types';
```

Comment:
Let's always put imports before exports. And new line between. At first I thought this was an import from where it was placed.

Reply:
Please add a space between the imports and exports.